docs/ops/convolution/DeformableConvolution_1.md

```diff
@@ -4,85 +4,180 @@
+  * **Default value**: explicit
   * **Required**: *no*
   * **Note**: *pads_begin* and *pads_end* attributes are ignored when *auto_pad* is specified.
 
+
 * *group*
 
   * **Description**: *group* is the number of groups which *output* and *input* should be split into. For example, *group* equal to 1 means that all filters are applied to the whole input (usual convolution), *group* equal to 2 means that both *input* and *output* channels are separated into two groups and the *i-th output* group is connected to the *i-th input* group channel. *group* equal to a number of output feature maps implies depth-wise separable convolution.
-  * **Range of values**: integer value starting from 1
+  * **Range of values**: integer value starting from `1`
   * **Type**: int
-  * **Default value**: 1
+  * **Default value**: `1`
   * **Required**: *no*
 
 * *deformable_group*
 
   * **Description**: *deformable_group* is the number of groups which deformable values and *output* should be split into along the channel axis. Apply the deformable convolution using the i-th part of the offset part on the i-th out.
-  * **Range of values**: integer value starting from 1
+  * **Range of values**: integer value starting from `1`
   * **Type**: int
-  * **Default value**: 1
+  * **Default value**: `1`
   * **Required**: *no*
 
 **Inputs**:
 
-*   **1**: Input tensor of rank 3 or greater. Required.
+*   **1**: Input tensor of type *T* and rank 3, 4 or 5. Layout is NCZYX (number of batches, number of channels, spatial axes Z, Y, X). Required.
+
+*   **2**: Deformable values tensor of type *T* and rank 3, 4 or 5. Layout is *TBD*. Required.
+
+*   **3**: Kernel tensor of type *T* and rank 3, 4 or 5. Layout is OIZYX (number of output channels, number of input channels, spatial axes Z, Y, X). Required.
+*   **Note**: Type of the convolution (1D, 2D or 3D) is derived from the rank of the input tensors and not specified by any attribute:
+      * 1D convolution (input tensors rank 3) means that there is only one spatial axis X
+      * 2D convolution (input tensors rank 4) means that there are two spatial axes Y, X
+      * 3D convolution (input tensors rank 5) means that there are three spatial axes Z, Y, X
+
 
-*   **2**: Deformable values tensor of rank 3 or higher. Required.
+**Outputs**:
 
-*   **3**: Convolution kernel tensor. Weights layout is OIYX (OIZYX for 3D convolution), which means that *X* is changing the fastest, then *Y*, then *Input* then *Output*. The size of kernel is derived from the shape of this input and not specified by any attribute. Required.
+*   **1**: Output tensor of type *T* and rank 3, 4 or 5. Layout is NOZYX (number of batches, number of kernel output channels, spatial axes Z, Y, X).
 
+**Types**:
+
+* *T*: Any floating point type.
+ 
 **Example**
 
+1D Convolution
+```xml
+<layer type="DeformableConvolution" ...>
+    <data dilations="1" pads_begin="0" pads_end="0" strides="2" auto_pad="valid" group="1" deformable_group="1"/>
```

Comment:
Why all the samples have 3 inputs? According to the specification there should be exactly 3 inputs.

Reply:
Yes it was a mistake. Fixed. I used MXNet framework to verify that we can execute DeformableConvolution with dimensions provided in examples.